Makefile

```diff
@@ -147,6 +147,16 @@ manifests.annotate:
 	sed '1,7d' $(SOURCE_DOCS_DIR)/reference/install.md > $(CROSSPLANE_CHART_DIR)/README.md
 	@$(OK) Copied and modified chart README.md from Crossplane docs
 
+local-kustomize:
+	@$(INFO) Adding all CRDs to Kustomize file for local development
+	@rm cluster/kustomization.yaml
```

Comment:
Should this be gitignored if we're deleting and regenerating when we run `make local-kustomize`?

Reply:
@negz do you mean the `cluster/kustomization.yaml`? `local-kustomize` is called as part of `make generate` so if there are differences then we will fail `check-diff`. We cannot `gitignore` this file because it is what we are adding to enable `kubectl apply -k` here.

Reply:
@negz could you circle back for another review here?